sources/tools/Stride.VisualStudio.Package/source.extension.vsixmanifest

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="248ff1ce-dacd-4404-947a-85e999d3c3ea" Version="4.0.6" Language="en-US" Publisher="Stride" />
-    <DisplayName>Stride</DisplayName>
-    <Description>Stride VisualStudio Package</Description>
+    <Identity Id="Stride.VisualStudio.Package.2022" Version="4.1.0" Language="en-US" Publisher="Stride" />
```

Comment:
Per https://docs.microsoft.com/en-us/visualstudio/extensibility/vsix-extension-schema-2-0-reference?view=vs-2022#metadata-element

Id - This attribute must be a unique ID for the package chosen by its author. The name should be qualified the same way CLR types are namespaced: Company.Product.Feature.Name. The Id attribute is limited to 100 characters.